### PR TITLE
Fix styling in widget forms in secondary tagging

### PR DESCRIPTION
**Pull request by subinasr**
## Changes

* Make controlled expandable container in Organigram Widget Form
* Fix styling in widget forms

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

File: app/views/AnalyticalFramework/FrameworkForm/components/WidgetEditor/OrganigramWidgetForm/index.tsx [hunk 1]
```diff
@@ -158,6 +162,14 @@ function NodeInput(props: NodeInputProps) {
         [onFieldChange, value?.children],
     );
 
+    const handleNodeExpansionChange = useCallback(
+        (nodeExpanded: boolean, nodeId: string) => {
+            setExpandedNodeId(nodeExpanded ? nodeId : undefined);
+            console.warn('I am inside handle change');
+        },
+        [],
+    );
```
Comment (tnagorra) [inline app/views/AnalyticalFramework/FrameworkForm/components/WidgetEditor/OrganigramWidgetForm/index.tsx]: Let's remove the console.warn that was meant for debugging

File: app/views/AnalyticalFramework/FrameworkForm/components/WidgetEditor/OrganigramWidgetForm/index.tsx [hunk 2]
```diff
@@ -175,22 +187,42 @@ function NodeInput(props: NodeInputProps) {
...
         onSiblingChange,
         onSiblingRemove,
         arrayError,
+        onExpansionChange,
+        expanded,
+        expandedNodeId,
     ]);
 
+    // console.warn(expandedNodeId, ' is ', expanded ? 'expanded' : 'not expanded');
+
```
Comment (tnagorra) [inline app/views/AnalyticalFramework/FrameworkForm/components/WidgetEditor/OrganigramWidgetForm/index.tsx]: Let's remove this comment

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/views/AnalyticalFramework/FrameworkForm/components/WidgetEditor/OrganigramWidgetForm/index.tsx [hunk 3]
```diff
@@ -158,6 +162,14 @@ function NodeInput(props: NodeInputProps) {
         [onFieldChange, value?.children],
     );
 
+    const handleNodeExpansionChange = useCallback(
+        (nodeExpanded: boolean, nodeId: string) => {
+            setExpandedNodeId(nodeExpanded ? nodeId : undefined);
+            console.warn('I am inside handle change');
```
Comment (samshara) [inline app/views/AnalyticalFramework/FrameworkForm/components/WidgetEditor/OrganigramWidgetForm/index.tsx]: remove console.warn

File: app/views/AnalyticalFramework/FrameworkForm/components/WidgetEditor/OrganigramWidgetForm/index.tsx [hunk 4]
```diff
@@ -175,22 +187,42 @@ function NodeInput(props: NodeInputProps) {
...
         autoFocus: newlyCreatedNodeIdRef.current === option.clientId,
         index: optionIndex,
+        onExpansionChange: handleNodeExpansionChange,
+        expanded: expandedNodeId === option.clientId,
     }), [
         onSiblingChange,
         onSiblingRemove,
         arrayError,
+        onExpansionChange,
+        expanded,
```
Comment (samshara) [inline app/views/AnalyticalFramework/FrameworkForm/components/WidgetEditor/OrganigramWidgetForm/index.tsx]:
```suggestion
      handleNodeExpansionChange,
```

**Review by samshara — APPROVED** [review]

**Review by AdityaKhatri — APPROVED** [review]

**Pull request merged**